src/main/java/io/jenkins/plugins/analysis/warnings/axivion/AxivionSuite.java

```diff
@@ -0,0 +1,240 @@
+package io.jenkins.plugins.analysis.warnings.axivion;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nonnull;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.auth.UsernamePasswordCredentials;
+
+import com.cloudbees.plugins.credentials.CredentialsMatchers;
```

Comment:
Where is this dependency coming from? Jenkins-core?

Reply:
It comes from the `Credentials Plugin` which is shipped by Jenkins and can't be deinstalled.